tests/e2e/tests/TimelineComponent/TimelineComponentDisabled.ts

```diff
@@ -31,7 +31,7 @@ export default (
       await userState.headerComponentIsVisible({
         countryName: NoTriggerDataSet.CountryName,
       });
-      await timeline.timelineIsInactive();
+      await timeline.timelineIsInactive(); // TODO: make conditional on disaster-type
```

Comment:
Are the ~~breadcrumbs~~ timelines inactive only used in flood?

We should use these features consistently for all country hazards. We can use conditional tests as a crutch until we refactor these exceptions out of the portal. In the long term, such branching of logic will only get more complex thus making IBF (and its tests) unmanageable.